### PR TITLE
OTG-4.8.17 - v4 migration cleanup

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR part of issue #84 

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix links
- Fix linting

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.17_Testing_for_HTTP_Incoming_requests_OTG-INPVAL-017.md [hunk 1]
```diff
@@ -1,41 +1,34 @@
-Summary
--------
+# Testing for HTTP Incoming requests
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.17_Testing_for_HTTP_Incoming_requests_OTG-INPVAL-017.md]: Could you please title cap this (as well as the filename.. and whatever else that might impact)

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.17_Testing_for_HTTP_Incoming_requests_OTG-INPVAL-017.md [hunk 2]
```diff
@@ -1,41 +1,34 @@
...
 This section describes how to monitor all incoming/outgoing http requests on both client or web server side. The purpose of this testing is to verify if there is unnecessary or suspicious http request sending in the background.
 
 Most of Web security testing tools (i.e. AppScan, BurpSuite, ZAP) act as Http Proxy. This will require changes of proxy on client side application or browser. The testing techniques listed below is primary focused on how we can monitor Http requests without changes of client side which will be more close to production usage scenario.
 
-Test Objectives
----------------
+## Test Objectives
 
-1\. Monitor all incoming and outgoing http requests to the Web Server to inspect any suspicious requests.
+1. Monitor all incoming and outgoing http requests to the Web Server to inspect any suspicious requests.
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.17_Testing_for_HTTP_Incoming_requests_OTG-INPVAL-017.md]: Remove the blank?

Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.17_Testing_for_HTTP_Incoming_requests_OTG-INPVAL-017.md]: @kingthorin Fixed

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.17_Testing_for_HTTP_Incoming_requests_OTG-INPVAL-017.md (hunk 1 above)
Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.17_Testing_for_HTTP_Incoming_requests_OTG-INPVAL-017.md]: @kingthorin Fixed

**Pull request merged**